modules/deploy/partials/high-availability.adoc

```diff
@@ -0,0 +1,376 @@
+
+ifdef::env-kubernetes[]
+Kubernetes offers an ideal environment for managing and scaling Redpanda. By deploying Redpanda on Kubernetes, you can ensure HA through Kubernetes-native features, benefit from automated rollouts and rollbacks, and take advantage of the platform's self-healing properties.
+endif::[]
+
+== Deployment strategies
+
+Consider the following Redpanda deployment strategies for the most common types of failures.
+
+|===
+| Failure | Impact | Mitigation strategy
+
+| Broker failure
+| Loss of function for an individual broker or for any virtual machine (VM) that hosts the broker
+| Multi-broker deployment
+
+| Rack or switch failure
+| Loss of brokers/VMs hosted within that rack, or loss of connectivity to them
+| Multi-broker deployment spread across multiple racks or network failure domains
+
+| Data center failure
+| Loss of brokers/VMs hosted within that data center, or loss of connectivity to them
+| Multi-AZ or replicated deployment
+
+| Region failure
+| Loss of brokers/VMs hosted within that region, or loss of connectivity to them
+| Geo-stretch (latency dependent) or replicated deployment
+
+| Global, systemic outage (DNS failures, routing failures)
+| Complete outage for all systems and services impacting customers and staff
+| Offline backups (Tiered Storage), replicas in 3rd-party domains
+
+| Data loss or corruption (accidental or malicious)
+| Corrupt or unavailable data that also affects synchronous replicas
+| Offline backups (Tiered Storage)
+|===
+
+ifndef::env-kubernetes[See also: xref:./production/production-deployment.adoc[Deploy for Production]]
+
+== HA deployment options
+
+This section explains the trade-offs with different HA configurations.
+
+* <<Multi-broker deployment>>
+* <<Multi-AZ deployment>>
+* <<Multi-region deployment>>
+* <<Multi-cluster deployment>>
+
+=== Multi-broker deployment
+
+Redpanda is designed to be deployed in a cluster that consists of at least three brokers. Although clusters with a single broker are convenient for development and testing, they aren't resilient to failure. Adding brokers to a cluster provides a way to handle individual broker failures. You can also use <<Rack awareness>> to assign brokers to different racks, which allows Redpanda to tolerate the loss of a rack or failure domain.
+
+ifdef::env-kubernetes[]
+By default, the Redpanda Helm chart deploys three Redpanda brokers in separate Pods that are all managed by a StatefulSet. If the Redpanda broker inside a Pod crashes, Kubernetes automatically restarts the Pod.
+
+To ensure Redpanda Pods are distributed across multiple nodes, the Redpanda Helm chart uses `podAntiAffinity` rules to discourage the scheduler from placing Redpanda Pods on the same node. This not only ensures that if a node crashes, you only lose one Redpanda broker, it's also essential because Redpanda brokers must have access to all resources on the node.
```

Comment:
I'm not really following the "it's also essential because Redpanda brokers must have access to all resources on the node" bit. Is it meant to convey that "since Redpanda brokers must have access to all resources on the node, Redpanda Pods should therefore not all be placed on the same node"? (And I stil don't understand the connection between these two ideas)